src/Generators/ModelGenerator.php

```diff
@@ -165,29 +159,48 @@ private function getPHPDocType($db_type, $relation = null, $relationText = null)
     {
         $relationText = (!empty($relationText)) ? $relationText : null;
 
-        switch ($db_type) {
+        switch (strtolower($db_type)) {
+            case 'increments':
+            case 'integer':
+            case 'smallinteger':
+            case 'long':
+            case 'biginteger':
+                return 'integer';
+            case 'double':
+            case 'float':
+            case 'real':
+            case 'decimal':
+                return 'float';
+            case 'boolean':
+                return 'boolean';
+            case 'string':
+            case 'char':
+            case 'text':
+            case 'mediumtext':
+            case 'longtext':
+            case 'enum':
+            case 'byte':
+            case 'binary':
+            case 'password':
+                return 'string';
+            case 'timestamp':
+            case 'date':
             case 'datetime':
                 return 'string|\Carbon\Carbon';
             case '1t1':
-                return '\\'.$this->commandData->config->nsModel.'\\'.$relation->inputs[0].' '.Str::camel($relationText);
+                return '\\' . $this->commandData->config->nsModel . '\\' . $relation->inputs[0] . ' ' . Str::camel($relationText);
             case 'mt1':
                 if (isset($relation->inputs[1])) {
                     $relationName = str_replace('_id', '', strtolower($relation->inputs[1]));
                 } else {
                     $relationName = $relationText;
                 }
-
-                return '\\'.$this->commandData->config->nsModel.'\\'.$relation->inputs[0].' '.Str::camel($relationName);
+                return '\\' . $this->commandData->config->nsModel . '\\' . $relation->inputs[0] . ' ' . Str::camel($relationName);
             case '1tm':
             case 'mtm':
             case 'hmt':
-                return '\Illuminate\Database\Eloquent\Collection'.' '.Str::camel(Str::plural($relationText));
+                return '\Illuminate\Database\Eloquent\Collection' . ' ' . Str::camel(Str::plural($relationText));
             default:
-                $fieldData = SwaggerGenerator::getFieldType($db_type);
-                if (!empty($fieldData['fieldType'])) {
```

Comment:
here in the default case, checking all fields types which are you added in the above cases

Reply:
No, the decimal field in the table returns a "number" type.
https://docs.phpdoc.org/latest/guides/types.html
We should get "float"

Reply:
public function initializeSoftDeletes()
    {
        $this->dates[] = $this->getDeletedAtColumn();
    }

https://github.com/laravel/framework/blob/6.x/src/Illuminate/Database/Eloquent/SoftDeletes.php#L34

Reply:
ok, I will check it.

Reply:
I checked it, so here you can update in `SwaggerGenerator::getFieldType()` method of `SwaggerGenerator.php` as per need instant of added all cases.

Reply:
I checked it.  here you can just update `SwaggerGenerator::getFieldType()` method of `SwaggerGenerator.php` as per needed instant of added all cases, already checked all cases in that function.